DDECal/test/tRotationConstraint.cc

```diff
@@ -48,11 +49,13 @@ BOOST_AUTO_TEST_CASE( test_rotation )
   }
 }
 
-BOOST_AUTO_TEST_CASE( test_rotation_and_diagonal )
+BOOST_DATA_TEST_CASE( test_rotation_and_diagonal,
+                      boost::unit_test::data::make({ true, false }),
+                      doPhaseReference )
 {
   RotationAndDiagonalConstraint constraint;
   constraint.InitializeDimensions(1, 1, 1);
-  constraint.SetDoPhaseReference(true);
+  constraint.SetDoPhaseReference(doPhaseReference);
 
   vector<vector<complex<double> > > onesolution(1);
```

Comment:
To test the constraint, you'll need at least two solutions (this dimension is the number of antennas).